lib/blockquote.jsx

```diff
@@ -0,0 +1,15 @@
+var React = require('react');
+
+var Blockquote = React.createClass({
```

Comment:
Since we're planning on reusing this across multiple pages, I think it would be useful to add some [prop validation](http://facebook.github.io/react/docs/reusable-components.html#prop-validation)... that helps self-document the code and will also make our smoke test raise errors if props don't validate at build time!